submarine-workbench/workbench-web/src/app/pages/workbench/experiment/experiment-home/experiment-home.component.html

```diff
@@ -17,5 +17,54 @@
  under the License.
 -->
 
-<!-- <p>experiment-home works!</p> -->
-<!-- TODO: should split home page from experiment root components -->
+<div style="margin: 15px; padding: 15px; background-color: white">
+  <div align="right">
+    <nz-radio-group>
+      <label nz-radio-button nzValue="All">All</label>
+      <label nz-radio-button nzValue="Own">Owned By Me</label>
+      <label nz-radio-button nzValue="Access">Accessible By Me</label>
+    </nz-radio-group>
+    <nz-input-group
+      nzSearch
+      style="width: 300px; margin-top: 15px; margin-left: 10px; margin-right: 5px"
```

Comment:
Is there any other method to avoid hard-coded width?

Reply:
I don't have a better solution, but I think it works quite fine now and is not a big problem.